server/verify.go

```diff
@@ -0,0 +1,112 @@
+package server
+
+import (
+	"bytes"
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/rsa"
+	"crypto/subtle"
+	"fmt"
+
+	"github.com/google/go-tpm-tools/client"
+	tpmpb "github.com/google/go-tpm-tools/proto"
+	"github.com/google/go-tpm/tpm2"
+)
+
+// Verify checks an AK's signature on a Quote against the quoted data and extra data.
+// Then, it matches that quoted data (PCR digest) against a group of PCRs.
+//
+// Verify supports ECDSA and RSASSA signature verification.
+func Verify(pubKey crypto.PublicKey, quote *tpmpb.Quote, pcrs *tpmpb.Pcrs, extraData []byte) error {
```

Comment:
Thoughts on VerifyQuote? I was thinking we may want a high level VerifyAttestation that encapsulates the PCRs, quotes, and event log.

Reply:
Ya this is good, and checks the right stuff.